extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -65,12 +65,6 @@ describes.realWin(
       'triggers': {'pageview': {'on': 'visible', 'request': 'foo'}},
     };
 
-    const noTriggersError =
-      '[AmpAnalytics <unknown id>] No triggers were ' +
-      'found in the config. No analytics data will be sent.';
-    const noRequestStringsError =
-      '[AmpAnalytics <unknown id>] No request ' +
-      'strings defined. Analytics data will not be sent from this page.';
```

Comment:
Do you think it necessary to check for these warnings in the tests?

Reply:
I think no. We had those checks because we throw errors with uncaught console error before. All those tests are not meant to test the empty trigger/request config.